waterbox/common.mak

```diff
@@ -44,7 +44,7 @@ ifneq ($(filter %.cpp,$(SRCS)),)
 EXTRA_LIBS := -lc++ -lc++abi -lunwind $(EXTRA_LIBS)
 endif
 
-_OBJS := $(addsuffix .o,$(realpath $(SRCS)))
```

Comment:
All changes look sane to me, but I'd just like to confirm with @nattthebear that this change does not break anything unexpectedly.

Reply:
Makefiles, huh.  Uhhhhh....

Make clean all and make all and see if it still works.  So long as an alligator doesn't come out, I'm inclined to say it's fine.

Reply:
Okay this breaks the `libco` makefile, possibly among others. This can be resolved by changing the `realpath` to an `abspath` call instead of removing it altogether.